CMakeLists.txt

```diff
@@ -1,17 +1,40 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.16.0)
+
+project(I3Pager)
 
 find_package(ECM REQUIRED NO_MODULE)
 
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR} ${CMAKE_MODULE_PATH})
 
+
 include(KDEInstallDirs)
 include(KDECMakeSettings)
 include(KDECompilerSettings)
+include(FeatureSummary)
+
+
+set(CMAKE_CXX_STANDARD 17)
+set(QT_MAJOR_VERSION "6")
+set(QT_MIN_VERSION "6.4.0")
+set(KF_MIN_VERSION "6.0.0")
+set(KF_MAJOR_VERSION "6")
```

Comment:
When compiling locally I had to move these above `KDEInstallDirs` otherwise I would get references to Qt5.

I'm far from a cmake expert, but it seems logical to move these to just below `project(I3Pager)` like done here: https://github.com/KDE/plasma-integration/blob/master/CMakeLists.txt#L7

Reply:
Makes sense. Changed.